kfs-core/src/main/java/edu/arizona/kfs/gl/service/impl/CollectorHelperServiceImpl.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * The Kuali Financial System, a comprehensive financial management system for higher education.
+ * 
+ * Copyright 2005-2014 The Kuali Foundation
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package edu.arizona.kfs.gl.service.impl;
+
+import edu.arizona.kfs.gl.service.GlobalTransactionEditService;
+import org.kuali.kfs.gl.batch.CollectorBatch;
+import org.kuali.kfs.gl.batch.CollectorStep;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.rice.krad.util.MessageMap;
+
+/**
+ * The base implementation of CollectorHelperService
+ *
+ * @see org.kuali.kfs.gl.batch.service.CollectorService
+ */
+public class CollectorHelperServiceImpl extends org.kuali.kfs.gl.batch.service.impl.CollectorHelperServiceImpl {
```

Comment:
Ha! This time from the opposite direction! I think the same tricky type of collapsed package illusion got me; I need to get my prescription glasses renewed, it's been too long apparently. I'll move this now and ping you on hipchat once it's pushed.